test/smoke/testApps/PerfTestAppSpringBoot/build.gradle

```diff
@@ -0,0 +1,49 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.19.RELEASE'
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
```

Comment:
Curious why you would need eclipse plug-in here?

Reply:
This was from a tutorial on how to get JSP to work in spring-boot/gradle.

Reply:
I see.